.travis.yml

```diff
@@ -0,0 +1,31 @@
+sudo: required
+
+language: scala
+
+services:
+- docker
+
+cache:
+  directories:
+  - $HOME/.ivy2/cache
+  - $HOME/.sbt/boot/
+
+script:
+  - /bin/true
```

Comment:
I think that is better than nothing. Probably should have chosen this for the other Scala project repositories.